pkg/controller/model/constants.go

```diff
@@ -17,46 +17,41 @@ package model
 
 import (
 	"time"
-
-	api "github.com/IBM/ibm-monitoring-grafana-operator/pkg/apis/operator"
 )
 
 const (
-	DefaultDSType                             = api.DSTypeCommonService
-	DefaultThanosURL                          = "https://thanos-querier.openshift-monitoring.svc:9091"
-	GrafanaConfigName                         = "grafana-config"
-	GrafanaLogVolumes                         = "grafana-log"
-	GrafanaDataVolumes                        = "grafana-storage"
-	GrafanaDatasourceName                     = "datasource-config"
-	GrafanaHealthEndpoint                     = "/api/health"
-	DefaultRouterPort                         = 8080
-	DefaultClusterPort                 int32  = 8443
-	GrafanaAdminSecretName                    = "grafana-secret"
-	GrafanaInitMounts                         = "grafana-init-mount"
-	GrafanaPlugins                            = "grafana-plugins"
-	GrafanaConfigMapsDir                      = "/etc/grafana-configmaps/"
-	GrafanaServiceAccountName                 = "ibm-monitoring-grafana"
-	GrafanaDeploymentName                     = "ibm-monitoring-grafana"
-	GrafanaServiceName                        = "ibm-monitoring-grafana"
-	GrafanaHTTPPortName                       = "web"
-	RequeueDelay                              = time.Second * 10
-	DefaultGrafanaPort                 int32  = 3000
-	GrafanaRouteName                          = "ibm-monitoring-grafana"
-	PrometheusServiceName              string = "ibm-monitoring-prometheus"
-	GrafanaAdminUserEnvVar                    = "username"
-	GrafanaAdminPasswordEnvVar                = "password"
-	ClusterDomain                             = "cluster.local"
-	PrometheusPort                     int32  = 9090
-	InitContainerName                         = "init-container"
-	DefaultInitImage                          = "quay.io/opencloudio/icp-initcontainer"
-	DefaultInitImageTag                       = "1.0.0-build.3"
-	DefaultDashboardControllerImage           = "quay.io/opencloudio/dashboard-controller"
-	DefaultDashboardControllerImageTag        = "v1.2.0-build.3"
-	DefaultBaseImage                          = "quay.io/opencloudio/grafana"
-	DefaultBaseImageTag                       = "v6.5.2-build.2"
-	DefaultRouterImage                        = "quay.io/opencloudio/icp-management-ingress"
-	DefaultRouterImageTag                     = "2.5.1"
-	DSProxyConfigSecName                      = "grafana-ds-proxy-config"
+	DefaultThanosURL                         = "https://thanos-querier.openshift-monitoring.svc:9091"
```

Comment:
Is this a documented URL by OCP and supported one? Perhaps should the URL be queried somehow, not hardcoded as it might possibly change in the never version of OCP?

Reply:
@pgodowski , i don't see any doc containing this, i see similar thing in this blog: https://www.openshift.com/blog/thanos-querier-versus-thanos-querier
We also have this mentioned in our ibm-grafana-ocpthanos-proxy repo(https://github.com/IBM/ibm-grafana-ocpthanos-proxy) readme file